tests/unit_tests/db/test_db_handler.py

```diff
@@ -33,11 +50,46 @@ def _db_cleanup(db, random_id):
     db.db_client[f"sandbox_{random_id}"]["sites"].drop()
 
 
+@pytest.fixture
+def test_db():
+    return "test_db"
+
+
+@pytest.fixture
+def test_file():
+    return "test_file.dat"
+
+
+@pytest.fixture
+def test_file2():
```

Comment:
Surprised the linters didn't complain about camel case (`test_file_2`)

Reply:
We have relaxed linting rules for the test functions.

Reply:
As you can see, I am not as relaxed as the linters...

Reply:
OK. Changed to `t_e_s_t_f_i_l_e_2.dat`